makefile.shade

```diff
@@ -34,11 +36,13 @@ use-standard-lifecycle
         Environment.Exit(-1);
         return;
       }
-  }
 
-  exec program='${Path.Combine(SANITY_CHECKER, "bin", "Release", "SanityCheck.exe")}' commandline='${BUILD_SHARE} ${BUILD_BRANCH} ${BUILD_DIR} ${SYMBOLS_DIR} ${NUGET_SYMBOLS_SHARE} ${SYMBOLS_NUGET_EXE} '
+   }
+
+  var args = '--drop-folder ${BUILD_SHARE} --build-branch ${BUILD_BRANCH} --output-path ${BUILD_DIR} --symbols-source-path ${SYMBOLS_DIR} --symbols-output-path ${NUGET_SYMBOLS_SHARE} --symbols-nuget-exe ${SYMBOLS_NUGET_EXE} --nuget-publish-feed ${NUGET_PUBLISH_FEED} --apikey ${APIKEY}'
+  exec program='cmd' commandline='/C dnx SanityCheck ${args}' workingdir='${SANITY_CHECKER}'
```

Comment:
Unrelated but we should rename the SanityCheck to something more descriptive because it does more than checking